specs/postgres/setup-postgres.sh

```diff
@@ -9,7 +9,7 @@ ACTION=$1
 STORAGE_CLASS_NAME=$2
 STORAGE_SIZE=$3
 NAMESPACE=$4
-DATABASE_DUMP_FILE_PATH=$5
+DATABASE_DUMP_FILE_PATH=$(realpath $5)
```

Comment:
Sorry this needs to be
```suggestion
DATABASE_DUMP_FILE_PATH=$5
if [[ $DATABASE_DUMP_FILE_PATH != s3://* ]]
then
   DATABASE_DUMP_FILE_PATH=$(realpath $5)
fi
```

Reply:
Updated as above.